sw/device/silicon_creator/mask_rom/BUILD

```diff
@@ -300,18 +300,26 @@ cc_test(
 # MASK ROM E2E TESTS
 
 # Bootup tests
-BOOT_FAILURE_MSG = "BFV:[0-9a-z]{{8}}\r\nLCV:[0-9a-z]{{8}}\r\n"
+BOOT_FAILURE_MSG = "BFV:[0-9a-f]{8}"
+
+BOOT_SUCCESS_MSG = "PASS!"
 
 opentitan_functest(
     name = "e2e_bootup_success",
     srcs = ["mask_rom_test.c"],
     cw310 = cw310_params(
         bitstream = "//hw/bitstream:mask_rom",
-        tags = ["broken"],
+        exit_failure = BOOT_FAILURE_MSG,
+        exit_success = BOOT_SUCCESS_MSG,
     ),
+    manifest = "//sw/device/silicon_creator/rom_ext:manifest_standard",
     signed = True,
     verilator = verilator_params(
+        exit_failure = BOOT_FAILURE_MSG,
+        exit_success = BOOT_SUCCESS_MSG,
         rom = ":mask_rom_sim_verilator_scr_vmem",
+        # Note: Leaving as broken since this test takes a long time and
+        # we don't really need it as long as the CI runs the FPGA version.
```

Comment:
@alphan, How long does this test take? I think it's one of the ones that I didn't investigate when I tried to update all the timeouts and I'm curious to know.

Reply:
@nmosier recently booted the current rom_ext on Verilator and said it took around ~30mins on his machine, so I would expect around that.

Reply:
Thanks! in the future we should mark things that take less than an hour ```timeout = "eternal"```

When tests take more than an hour we should set the timeout to eternal and add a comment that describes how eternal should be interpreted to reliably run the test.

Reply:
Fixed in #14627